fiftyone/operators/__init__.py

```diff
@@ -19,6 +19,7 @@
 )
 from .utils import ProgressHandler, is_new
 from .panel import Panel, PanelConfig
+from .store import ExecutionStore
```

Comment:
By convention, I prefer that anything that we're publicly documenting for users to be accessible via the highest reasonable package level. For example in this case:

```py
# happy
from fiftyone.operators import ExecutionStore

# sad; too low-level for end users
from fiftyone.operators.store import ExecutionStore
```